packages/@aws-cdk/aws-lambda-python/lib/function.ts

```diff
@@ -0,0 +1,103 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as cdk from '@aws-cdk/core';
+import { Bundling } from './bundling';
+import { parseStackTrace } from './util';
+
+/**
+ * Properties for a PythonFunction
+ */
+export interface PythonFunctionProps extends lambda.FunctionOptions {
+  /**
+   * Path to the (Python) entry file.
+   *
+   * @default - Derived from the name of the defining file and the construct's id.
+   * If the `PythonFunction` is defined in `stack.ts` with `my-handler` as id
```

Comment:
How would that behave in a CDK python app? Will it use the defining python file name.
Perhaps just make this required to avoid too much magic.